src/act.ts

```diff
@@ -1,7 +1,101 @@
-import { act } from 'react-test-renderer';
+import { act as reactTestRendererAct } from 'react-test-renderer';
+import { checkReactVersionAtLeast } from './checkReactVersionAtLeast';
 
 const actMock = (callback: () => void) => {
   callback();
 };
 
-export default act || actMock;
+type GlobalWithReactActEnvironment = {
```

Comment:
Can we use TS `declare` to extend global object? https://bobbyhadz.com/blog/typescript-declare-global-variable#declare-global-variables-in-typescript